training/templates/training/error.html

```diff
@@ -2,11 +2,20 @@
 
 {% block content %}
 
-<div class="errormessage">
-    Error {{ message }}	
-    <br />
-    <a href="https://{{ host }}">Return to Galaxy</a>
-</div>
+<div class="container">
+	<div class="errormessage">
+		<p class="alert alert-danger">
+			Sorry, the site encountered an error
+		</p>
+
+		<br>
+
+		{{ message }}
 
+		<br>
+
+		<a href="/">Return to Galaxy</a>
```

Comment:
This may not always be correct, that was the reason for using `{{ TIAAS_DOMAN }}`, if they've prefixed their galaxy with a proxy prefix, e.g. `/galaxy`/ then this breaks.

Reply:
Ahh ok didn't appreciate the context there, thanks. I think that also explains the static files question.
Will fix.